filterdata/FilterTree4ML.py

```diff
@@ -22,7 +22,7 @@
 parser = argparse.ArgumentParser(description='Arguments')
 parser.add_argument('configfile', metavar='text', default='cfgFileName.yml',
                     help='input config yaml file name')
-parser.add_argument('--parquet', default=False, action='store_true',
+parser.add_argument('--parquet', default=True, action='store_true',
```

Comment:
In this way don't we save always to a parquet? Even if `--parquet` is not specified? https://docs.python.org/3/library/argparse.html#default

Reply:
Indeed you are right. I wanted to put parquet as default since in the end we use it and not root most of the time, but it was not working like this. In 
ec210d4 I changed the argument from `--parquet` to `--root` so that now parquet is the default and if you want to save the output in root you have to enable it. Would it be ok for you?

Reply:
OK